README.md

```diff
@@ -100,3 +100,11 @@ After 10 minutes without hitting the API the results will be:
 - 31th request - 10 seconds delay
 
 \*Delay remains the same because the value of `maxDelay` option is `100 seconds`
+
+
+## Integration
+
+This plugin is compatible and it can be integrated with other fastify plugins, but pay attention when using with [fastify-rate-limit](https://github.com/fastify/fastify-rate-limit)
+
+In this case, the `max` option should be greater than the `delayAfter` option used. Thereby, the delay for the response will be applied before the rate limit.
```

Comment:
let's clarify this better. the first sentence is pretty obvious, of course this plugin can be used with other plugins. this section should be specifically about the integration with rate-limit and should explain:

1. why you may want to do it
2. how to do it in terms of proper configuration
3. an example to be included in the repo and linked from here
4. caveats to avoid